### DISCUSSION TITLE
Specific licensing question (site for a content creator)

### DISCUSSION THREAD

**Discussion by ribbanya** [statiqdev/Discussions#70] [Q&A] [answered]
Hi, I'm currently (pro bono) developing a static site for a small YouTube content creator.

The nature of her content is educational, and her content is free. The website will be free to access. However, the creator is funded voluntarily through Patreon and her videos are probably monetized, so I'm not sure if that removes this purpose from "personal use." Similarly, neither of us are part of something so big as an "educational institution." I feel that this project is in the _spirit_ of the Prosperity License, but could you please confirm whether or not this falls into its purview?

Thank you!

**Comment by daveaglick** [comment, marked as answer]
Yeah, you're fine. I take a really loose interpretation of the license and would even like to get something in there about minimum revenue before the license clause kicks in (still looking for a good open source license that's similar to Prosperity with such an addition). In any case, I'm not interested in enforcing licensing for very small/educational/etc. personal uses.

**Reply by ribbanya** [reply]
Got it, thanks!

To be honest, I assumed it wouldn't be enforced either way, but I don't want to use someone's library without their permission.

Anyway, it looks great so I'm excited to start working with it.

**Reply by daveaglick** [reply]
Awesome, let me know how you get along!

**Reply by ror3d** [reply]
I was looking at the license issues about it and there seems they link to some alternative that might give this kind of permission - https://github.com/licensezero/prosperity-public-license/issues/6

My main doubt here was that one could consider using the software to build some kind of portfolio in breach of the commercial-use clause. I suspect from your comment here that you would not consider it so, but I don't think the license itself addresses this kind of use very much.

_edit: okay, after looking at other threads, I see that personal portfolio is considered fair use in this case. Awesome! thank you very much._

**Reply by daveaglick** [reply]
Yep! A portfolio is totally fine.

I’ve also been keeping a very close eye on both Polyform Small Business and some other similar licenses to help clarify and open the project to very small revenue generation without fees (I.e. showing a couple ads to earn a little movie money is not really my intent of the commercial clause). Not sure when/if I’ll actually change it since I’ve been able to clarify intent here when there’s questions, but worth considering.
